Models/User.js

```diff
@@ -0,0 +1,35 @@
+const { Schema, model } = require('mongoose');
+///////////////////////////////
+// MODELS
+////////////////////////////////
+const VideoidSchema = new Schema(
+  {
+  URL: String,
+  log: String,
+  },
+    { timestamps: true }
+);
+
+const ObjectidSchema = new Schema(
+  {
+  Time: String,
+  Geoposition: String,
+  isActive: Boolean,
+  video: [Videoid],
+  },
+    { timestamps: true }
+);
+
+const UserSchema = new Schema(
+  {
+  userid: String,
+  name: String,
+  email: String,
+  log: [Objectid],
+  },
+    { timestamps: true }
+);
+
+const User = model("User", UserSchema);
+
+module.exports = User;
```

Comment:
and we also have to export all of our models to do that let's export an object with the User, Video, and Log models in it so something like this.

`
module.exports = {User, Video, Log}
`